base/ca/shared/conf/CS.cfg

```diff
@@ -673,6 +673,7 @@ dbs.replicaDN=ou=replica
 dbs.replicaRangeDN=ou=replica, ou=ranges
 dbs.ldap=internaldb
 dbs.newSchemaEntryAdded=true
+dbs.numberRangeRadix=-1
```

Comment:
Is `dbs.numberRangeRadix` needed for migration? If not I'd suggest we drop it from the PR or move it into a separate PR. I also think that we probably need separate params such as `dbs.request.id.radix` and `dbs.cert.id.radix` to replace the hard-coded radixes in `RequestRepository` and `CertificateRepository`.

Reply:
Yes, I am implementing the CLI for update and I also thought to remove `dbs.numberRangeRadix`. I have added this  because I was thinking to fix without a new generator but with the new generator is better to remove.

Currently the hard coded is only used to write back the numbers in CS.cfg but for reading it is used the format, if with prefix `0x` it is hex otherwise it is decimal. This should not generate problems so I think to not add the option to write in a specific base.  Should the radix be configurable?

Reply:
It's not required to be configurable, but it could help to show what radix it's using now, and in case we want to change the radix the admin will have the full control of when to change it.

Reply:
Done! I have replaced with the two parameters but I have not added them to the default yet. Not sure if we should include.

Reply:
Thanks for the update!